Description: Fixes the compatibility with the version of Ant in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/src/classycle/ant/DependencyCheckingTask.java
+++ b/src/classycle/ant/DependencyCheckingTask.java
@@ -150,7 +150,7 @@
       Analyser analyser = new Analyser(getClassFileNames(), getPattern(), 
                                        getReflectionPattern(), 
                                        isMergeInnerClasses());
-      Map<Object, Object> properties = _definitionFile == null ? getProject().getProperties() 
+      Map properties = _definitionFile == null ? getProject().getProperties() 
                                                : System.getProperties();
       DependencyChecker dependencyChecker 
                         = new DependencyChecker(analyser,
